docs/source/contributing.rst

```diff
@@ -115,3 +115,105 @@ Integration tests
 * It should run correctly when used in Firefox on RHEL/CentOS 7 (this
   requirement is not currently checked in the CI environment, but can be
   tested locally by running `behave --wip`)
+
+Setting up a development environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This guide assumes that you are running Fedora. Make sure to translate the commands if you are running a different distribution.
```

Comment:
@mfranczy Agreed - I think however in this particular case it's just good enough